compose.dev.yml

```
@@ -33,11 +47,73 @@ services:
       - ./senderverifmilter/:/app
     command: bash -c "pip install --user -r requirements.txt ; python -u main.py"
 
-  watcher:
-   image: node:18-alpine
-   user: ${UID}:${GID}
-   working_dir: /app
-   volumes:
-        - ./app:/app
-   restart: unless-stopped
-   command: /bin/sh -c "yarn install && yarn watch"
+  mailpit:
+    image: axllent/mailpit:v1.22
+    ports:
+      - $MAILPIT_WEB:8025
```

Je trouve ça bien d’avoir des valeurs par défaut quand on peut, ça évite un peu de config superficielle côté dev:

```suggestion
      - ${MAILPIT_WEB_PORT:-8025}:8025
```

---

Ah, j'en avais viré pleins au début :smile: 
Je trouve que ça charge inutilement les compose file, décommenter une variable dans l'env c'est pas grand chose et ça force a voir rapidement ce que tu utilises. Et puis dans la doc, quand on parle de valeur par défaut ça peux porter à confusion, entre la valeur commenté dans le .env et celle dans le compose file
Puis tu va toujours tomber sur la personne qui modifie son docker-compose pour changer cette valeur au lieu du .env … bref perso j'aime pas trop !

---

Ok, ça marche !